crates/bpe/tests/src/lib.rs

```diff
@@ -1,16 +1,3 @@
-use bpe::byte_pair_encoding::BytePairEncoding;
-use rand::{thread_rng, Rng};
-
-pub fn create_test_bytes(bpe: &BytePairEncoding, tokens: usize) -> Vec<u8> {
```

Comment:
I "think" this function was testing more corner cases than the `create_test_string` variation 🤷

Reply:
Probably true, yes. I'll see where I can reinstate that.